aten/src/ATen/native/transformers/attention.cpp

```diff
@@ -682,9 +682,9 @@ std::tuple<Tensor, Tensor> _scaled_dot_product_attention_math(
     attn = at::softmax(attn, -1);
     if (dropout_p > 0.0) {
       if (dropout_mask.has_value()) {
```

Comment:
Maybe add a comment here that this isn't meant to be used by anyone directly? Or perhaps even raise a warning that this is for testing only?

Just in case someone attempts to use this in some other part of the codebase (however low the chances).

Reply:
great idea!